### ISSUE TITLE
Fix Logos and add features to Node Map

### ISSUE THREAD

**Issue by AstroWa3l** [armada-alliance/armada-alliance#20]
We should fix a few things and add a few things in the next update to our node data fields on the website

- Logos on some members does not appear, not sure why since images seem to work on adapools.org, so we should fix this
- Name
- Metrics Endpoint
- Location
- Minted Blocks
- Pool size (delegation/pledge)

**Comment by wcatz** [comment]
Map pins are too big. A nice little green pinging dot with a click reveal. This will also obscure the issue with thumbnails not being displayed at first glance.

**Comment by sublayerio** [comment]
@AstroWa3l 

Images are now fixed. Images have to be pulled and saved in the repo in order to be displayed on canvas (running into a cors issue otherwise).

@wcatz I've decided to not display the logo's on the map for the following reasons: Pointers were too big, the more pools the less visible the individual ones become. I tried clustering but this hides pools when zoomed out and makes it seem like we have less pools. I've kept it simple for now since the focus has to be on other features first. We can always improve the map along the way.
